PWGHF/DataModel/HFCandidateSelectionTables.h

```diff
@@ -34,9 +34,15 @@ DECLARE_SOA_TABLE(HFSelD0Candidate, "AOD", "HFSELD0CAND", //!
 namespace hf_selcandidate_dplus
 {
 DECLARE_SOA_COLUMN(IsSelDplusToPiKPi, isSelDplusToPiKPi, int); //!
+DECLARE_SOA_COLUMN(IsRecoHFFlag, isRecoHFFlag, int);           //!
```

Comment:
Maybe this flag is redundant, i.e. we want to process only the candidates that are flagged as Dplus from the candidate creator. Moreover, the if  at https://github.com/AliceO2Group/O2Physics/blob/2a17f74e16478e84ba55fb0e5bf402bfc9757541/PWGHF/Tasks/taskDPlus.cxx#L77-78 should be removed now that we can use binary operations in filters (it should be possible with the new arrow).